.github/workflows/ci.yml

```
@@ -238,11 +238,23 @@ jobs:
           checks/requirements.py
 
   notebook:
-    name: Notebook check
     env:
-      SUPERSTAQ_API_KEY : ${{ secrets.SUPERSTAQ_API_KEY }}
-      IBMQ_TOKEN : ${{ secrets.IBMQ_TOKEN }}
+      SUPERSTAQ_API_KEY: ${{ secrets.SUPERSTAQ_API_KEY }}
+      IBMQ_TOKEN: ${{ secrets.IBMQ_TOKEN }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - type: "qiskit-superstaq"
+            pattern: "*_qss.ipynb"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq[dev] -e ./qiskit-superstaq[dev]"
```

imo we should also remove the actions/cache... block from these tests and change this to
```suggestion
            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples]"
```
so this doubles as a test of our example requirements (and hopefully catches things like #885)